rdt-open-reader/src/main/java/io/ona/rdt/viewholder/PatientRegisterViewHolder.java

```diff
@@ -57,9 +59,19 @@ public void getView(Cursor cursor, SmartRegisterClient client, RegisterViewHolde
         String patientId = Utils.getValue(commonPersonObjectClient.getColumnmaps(), Constants.DBConstants.PATIENT_ID, true);
         String nameAndAge = createNameAndAgeLabel(patientName, patientId, patientAge);
 
-        final Patient patient = new Patient(patientName, sex, baseEntityId, patientId);
-        viewHolder.tvPatientNameAndAge.setText(nameAndAge);
-        viewHolder.tvPatientSex.setText(sex);
+        if(patientName.contains("coba")){
```

Comment:
This is implementation-specific information. It should ideally be part of a flavor.

Also, previous functionality should be maintained.